web-ui/src/components/menu/Menu.jsx

```diff
@@ -1,11 +1,20 @@
 import React, { useContext, useEffect, useRef, useState } from "react";
 
 import { postEmployeeHours } from "../../api/hours";
-import {reportAllMembersCsv} from "../../api/member"
```

Comment:
Most of the changes here are from Prettier. I'll add more comments to describe all the non-Prettier changes.